frontend/public/components/utils/kebab.tsx

```diff
@@ -196,7 +205,7 @@ export type KebabOption = {
   href?: string, callback?: () => any;
   accessReview?: AccessReviewResourceAttributes;
 };
-export type KebabAction = (kind: K8sKind, obj: K8sResourceKind) => KebabOption;
+export type KebabAction = (kind: K8sKind, obj: K8sResourceKind, volume?: any) => KebabOption;
```

Comment:
We should avoid changing the generic kebab action type for this one case if possible. If would be better to have a specific type for the volumes table.